apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/tips/tips.js

```diff
@@ -0,0 +1,79 @@
+
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
```

Comment:
Is this available on WP 5.4 (on all Atomic sites) or would you need to check it's not undefined?

Reply:
I'm checking instance below.

Reply:
> Is this available on WP 5.4 (on all Atomic sites)

I don't think so btw :-)